cookbook/request/load_balancer_reverse_proxy.rst

```diff
@@ -57,9 +62,9 @@ the IP address ``192.0.0.1`` or matches the range of IP addresses that use
 the CIDR notation ``10.0.0.0/8``. For more details, see the
 :ref:`framework.trusted_proxies <reference-framework-trusted-proxies>` option.
 
-That's it! Symfony will now look for the correct ``X-Forwarded-*`` headers
-to get information like the client's IP address, host, port and whether or
-not the request is using HTTPS.
+That's it! Symfony will now look for the correct headers to get information
+like the client's IP address, host, port and whether or not the request is
```

Comment:
Yeah, I agree with your comment here :) It was a response to "and in any other usage on symfony docs", as that has to be done with care (otherwise, someone will run grep and we have all usages changed)